docs/Call_for_papers_Bioinformatics.md

```diff
@@ -0,0 +1,132 @@
+# SciPyLA 2017
+## Call for Bioinformatics Research Papers
+
+![Alt text](../logo/scipyla2017.png)
+
+## Mayo 2017, La Habana, Cuba
+
+[Sitio Oficial ScipyLA 2017](http://scipyla.org/conf/2017/)
+
+### "Más python, más ciencia"
+
+5ta Conferencia Latinoamericana de Computación Cienfítica con Python (SciPyLA 2017), La Habana, Cuba, 10 de Mayo del 2017.
+
+## Taller Latinoamericano de Python y Bioinformática
+
+SciPyLA 2017 será la ocasión para explorar oportunidades de innovación y conexión con la industria biotecnológica. Cuba destaca dados sus 893 permisos comerciales registrados en más de 50 países. Conozca de primera mano cómo la biotecnología se abre camino en la región.
+
+## Objetivos
+
+*   Promover el uso de enfoques accesibles y reproducibles en la investigación científica.
+*   Presentar adelantos científico-técnicos recientes utilizando el lenguaje de programación Python o herramientas hechas en -o relacionadas con- Python.
+*   Promover el uso del lenguaje de programación Python en las actividades de investigación científica.
+*   Contribuir con el desarrollo tecnológico de Cuba y América Latina.
+
+Ayúdenos a organizar sus intereses en la agenda de la conferencia enviando su área específica de investigación a los [contactos del evento](#Contact). Gracias a la contribución de investigadores internacionales y de Cuba usted puede conocer nuestra [lista actual de tópicos del evento](./Tracks.html).
```

Comment:
I am not sure of whether the phrase *Ayúdenos a organizar sus intereses en la agenda de la conferencia enviando ...* is correct . I'd rather prefer *Con el fin de ofrecerle una experiencia inclusiva durante la conferencia, el equipo organizador quisiera conocer los aspectos que motivarían su participación. Usted puede enviar ...* .